lms/templates/commerce/receipt.underscore

```diff
@@ -1,7 +1,7 @@
 <div class="wrapper-content-main payment-confirmation-step">
   <article class="content-main">
     <h3 class="title">
-      <%= gettext( "Thank you! We have received your payment." ) %>
+      <%= gettext( "Thank you! We have received your payment for <span class='course_name_placeholder'></span>" ) %>
```

Comment:
Small i18n issue: please interpolate the `<span>` tag into the string.  For example:
```
interpolate( gettext( "Thank you!  We have received your payment for %s" ), "<span></span>")
```

The reason is that translators don't necessarily know HTML, so if we include the `<span>` tags in the translation string, translators might change the tag so it displays incorrectly.